skll/config.py

```diff
@@ -988,20 +976,22 @@ def _parse_and_validate_metrics(metrics, option_name, logger=None):
     # and parse it correctly
     metrics = yaml.safe_load(_fix_json(metrics))
     if not isinstance(metrics, list):
-        raise TypeError("{} should be a list, not a {}.".format(option_name, type(metrics)))
+        raise TypeError("{} should be a list, not a {}.".format(option_name,
+                                                                type(metrics)))
 
-    # `mean_squared_error` should be replaced with `neg_mean_squared_error`
+    # `mean_squared_error` is no more supported.
+    # It is replaced by `neg_mean_squared_error`
     if 'mean_squared_error' in metrics:
-        logger.warning("The metric \"mean_squared_error\" "
-                       "is deprecated and will be removed in the next "
-                       "release, please use the metric "
-                       "\"neg_mean_squared_error\" instead.")
-        metrics[metrics.index('mean_squared_error')] = 'neg_mean_squared_error'
+        raise ValueError("The metric \"mean_squared_error\" "
```

Comment:
You are repeating "is". 

Again, go shorter: 

"The metric \"mean_squared_error\" is no longer supported. Please use ..."

Reply:
fixed.